src/java/devcontainer-feature.json

```diff
@@ -59,6 +59,21 @@
       ],
       "default": "latest",
       "description": "Select or enter a Maven version"
```

Comment:
Can we update the minor version in here?
https://github.com/devcontainers/features/blob/21b034ed57a0b3c09ddd74e0995fe884c7566a28/src/java/devcontainer-feature.json#L3